VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -53,114 +46,83 @@ final class FriendListTableViewController: UITableViewController, UIViewControll
                 for: indexPath
-            ) as? CommonGroupTableViewCell else { return UITableViewCell() }
+            ) as? CommonGroupTableViewCell
+        else { return UITableViewCell() }
 
-        guard usersAtSections.count > 1 else { return UITableViewCell() }
-        cell.configureCell(unit: usersAtSections[indexPath.section][indexPath.row], labelNameTapHandler: tapHandler)
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        let key = sortedKeys[indexPath.section]
+        guard
+            let friendsListSection = sortedFriendsMap[key],
+            let friendsInfo = friendsListSection[safe: indexPath.row]
+        else { return UITableViewCell() }
+        cell.configureCell(unit: friendsInfo, labelNameTapHandler: tapHandler)
         return cell
     }
 
     // MARK: - Private methods
 
-    private func clearAllArrays() {
-        users.removeAll()
-        sortedUsers.removeAll()
-        lettersCountMap.removeAll()
-        sortedlettersMap.removeAll()
-        usersAtSections = [[]]
-        lettersMap = Alphabets.russianLettersMap
+    private func configureScreen() {
+        fetchFriends()
+        setupScene()
+    }
+
+    private func fetchFriends() {
+        NetworkService.shared.fetchFriends { friends in
+            let array = (friends as? ResponseWithFriends)?.response.items.map { friend in
+                NetworkUnit(
+                    name: "\(friend.firstName) \(friend.lastName)",
+                    description: friend.city?.title ?? Constants.emptyString,
+                    avatarImageName: friend.photo ?? Constants.emptyString,
+                    unitImageNames: [],
+                    id: friend.id
+                )
+            }
+            self.makeFriendsSortedMap(friendsInfo: array ?? [])
+            self.tableView.reloadData()
+        }
     }
 
-    private func configScreen() {
-        configDotsLoader()
+    private func setupScene() {
         regCells()
-        configreTapHandler()
-    }
-
-    private func setupCell() {
-        makeUsers()
-        getFirstLettersCount()
-        calculateNumberOfSections()
-        sortLettersCountMap()
-        putContactsInSections()
+        configureTapHandler()
     }
 
-    private func makeUsers() {
-        var indexCounter = 0
-        for user in UserNames.names {
-            users.append(NetworkUnit(
-                name: user,
-                description: UserLocations.locations[safe: indexCounter] ?? "",
-                avatarImageName: UserAvatarImageNames.avatarImageNames[safe: indexCounter] ?? "",
-                unitImageNames: UserAvatarImageNames.avatarImageNames
-            ))
-            indexCounter += 1
-        }
-    }
-
-    private func getFirstLettersCount() {
-        sortedUsers = users.sorted(by: { user1, user2 in
+    private func makeFriendsSortedMap(friendsInfo: [NetworkUnit]) {
+        var friendsMap: [Character: [NetworkUnit]] = [:]
+        friendsInfo.forEach { info in
```

Comment:
```suggestion
        friendsInfo.forEach { 
        guard let key = $0.name.first else { return }
```
вот так вот $0 можно?